MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -727,6 +739,11 @@ - (void)deleteCrashReportWithFileURL:(NSURL *)fileURL {
   NSError *error = nil;
   if ([fileURL checkResourceIsReachableAndReturnError:&error]) {
     [self.fileManager removeItemAtURL:fileURL error:&error];
+    if (error) {
+      MSLogError([MSCrashes logTag], @"Error deleting file \"%@\": %@", fileURL, error.localizedDescription);
+    }
+  } else if (error) {
```

Comment:
Just else. I wonder we need this log here in this case. If we can remove this else-if statement, simply pass nil to error in if condition.